lets-encrypt/activate-ssl.sh

```diff
@@ -242,7 +245,7 @@ fi
 # Check if $certfiles exists
 if [ -d "$certfiles" ]; then
 # Activate new config
-        bash /var/scripts/test-new-config.sh
+        bash /var/scripts/test-new-config.sh $domain.conf
```

Comment:
Proof of work: http://imgur.com/a/sDALF

I made two gist with the script so you can test it.
https://gist.github.com/Pe46dro/a6e26ce6c3bc6d0a70ff8053d80b5a3c
https://gist.github.com/Pe46dro/c9c1e84c1778f733d54d5954bb7cdd81